.github/workflows/Test.yml

```diff
@@ -5,29 +5,16 @@ on:
     branches:
       - main
     tags: ["*"]
-  pull_request:
-    branches:
-      - main
-    paths:
-      - "src/**"
-      - "test/**"
-      - "template/**"
-      - "*.toml"
-      - "copier.yml"
-    types: [opened, synchronize, reopened]
   workflow_dispatch:
 
-concurrency:
-  # Skip intermediate builds: always.
-  # Cancel intermediate builds: only if it is a pull request build.
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
-
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
-    runs-on: ${{ matrix.os }}
-    continue-on-error: ${{ matrix.allow_failure }}
+    uses: ./.github/workflows/ReusableTest.yml
+    with:
+      os: ${{ matrix.os }}
+      version: ${{ matrix.version }}
+      arch: ${{ matrix.arch }}
+      allow_failure: ${{ matrix.allow_failure }}
```

Comment:
I would move the `uses:` block below the `matrix` strategy.

Reply:
I like it first thing.